Gemfile

```diff
@@ -5,10 +5,5 @@ source 'https://rubygems.org'
 # development dependencies will be added by default to the :development group.
 gemspec
 
-# Declare any dependencies that are still in development here instead of in
-# your gemspec. These might include edge Rails or gems from your path or
-# Git. Remember to move these dependencies to your gemspec before releasing
-# your gem to rubygems.org.
-
-# To use a debugger
-# gem 'byebug', group: [:development, :test]
+# Load Gemfile with dependencies from manageiq
+eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
```

Comment:
We should probably create `spec/` directory in this plugin so that we can create the `spec/manageiq` symlink nicely, otherwise if I create the directory manualy, it will show as untracked by git.

Reply:
Good point. Done.